addon/components/paper-button.js

```diff
@@ -7,14 +7,22 @@ import ColorMixin from 'ember-paper/mixins/color-mixin';
 const { computed } = Ember;
 
 export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
-  attributeBindings: ['type'],
   type: 'button',
-  tagName: 'button',
+  tagName: computed('href', function() {
```

Comment:
But you're right. A computed without a dependent key should not raise a deprecation, I think. We should PR ember core. :)

Yak shaving.